### DISCUSSION TITLE
beaker:// urls only small subset of chrome:// urls

### DISCUSSION THREAD

**Discussion by Alex313031** [beakerbrowser/beaker#1887] [Q&A]
beaker:// can only access a few of the usual chromium-based-browser chrome (or equivalent) :// urls.
I would like to be able to access chrome://flags, chrome://gpu, chrome://memory, and chrome://version for instance. I know that beaker is not built off of full chromium source code, only electron and blink, so is that to blame? Will this be implemented later? @pfrazee @poga @edrex @rocketbear27 #

**Comment by rmlrml** [comment]
I would also like to be able to access beaker://flags to turn off smooth scrolling
